HW12/hw_12_task_2.py

```diff
@@ -0,0 +1,13 @@
+# 2. (необов'язкове виконання) Написати власний менеджер контексту, задачею якого буде друкувати "==========" – 10 знаків
+# дорівнює перед виконанням коду та після виконання коду, таким чином виділяючи блок коду символами дорівнює.
+
+class MyContextManager:
+    def __enter__(self):
+        print("=" * 10)
+        return 1
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("=" * 10)
```

Comment:
nit: exception can be processed here like:
```suggestion
        print("=" * 10)
        if exc_val:
            print(exc_val)
            return True
```